runtime/doc/lua.txt

```diff
@@ -2370,4 +2370,19 @@ read({path})                                               *vim.secure.read()*
         (string|nil) The contents of the given file if it exists and is
         trusted, or nil otherwise.
 
+trust({path}, {mode})                                     *vim.secure.trust()*
+    Update the trust status of file at {path} in the trust database at
+    $XDG_STATE_HOME/nvim/trust.
+
+    Parameters: ~
+      • {path}  (string) Path to a file to update status for.
+      • {mode}  (string) One of the following:
+                • 'allow': Add file to trust database and set it as trusted
+                • 'deny': Add file to trust database and set it as denied
+                • 'forget': Remove file from trust database
```

Comment:
nit: any reason not to use more common wording? `remove` / `delete`  
```suggestion
                • "remove": Remove file from trust database
```

Reply:
Resolved in 2fe2e19ee005d228b89bad38ac711df37adbbdc3